src/crab/ebpf_checker.cpp

```
@@ -344,8 +344,32 @@ void EbpfChecker::operator()(const ValidAccess& s) const {
             break;
         }
         case T_CTX: {
-            const auto ctx_size = context.program_info().type.ctx_descriptor->size;
+            const auto* desc = context.program_info().type.ctx_descriptor;
             const auto [lb, ub] = lb_ub_access_pair(s, reg.ctx_offset);
```

Cppcheck (2.20.0)

[style] The function 'unsigned_high' is never used.

(unusedFunction)